src/multibody/liegroup/special-orthogonal.hpp

```diff
@@ -383,7 +383,7 @@ namespace pinocchio
       assert(quaternion::isNormalized(quat1,RealScalar(PINOCCHIO_DEFAULT_QUATERNION_NORM_TOLERANCE_VALUE)));
       
       PINOCCHIO_EIGEN_CONST_CAST(Tangent_t,d)
-        = log3((quat0.matrix().transpose() * quat1.matrix()).eval());
+        = log3((quat0.conjugate() * quat1).matrix().eval());
```

Comment:
```suggestion
        = quaternion::log3(quat0.conjugate() * quat1);
```

Reply:
This should save some precision roundings.

Reply:
do we need the `.eval()`? 
like `quaternion::log3((quat0.conjugate() * quat1).eval());`